src/storage/redis_db.cc

```diff
@@ -578,9 +579,20 @@ rocksdb::Status SubKeyScanner::Scan(RedisType type, const Slice &user_key, const
       break;
     }
     InternalKey ikey(iter->key(), storage_->IsSlotIdEncoded());
+    auto value = iter->value().ToString();
+    // filter expired hash feild
```

Comment:
Typo in the same word. `feild`. I also tend to make typos in the word `field` :)